src/compas_timber/ghpython/components/CT_Beam_fromCurve/code.py

```diff
@@ -6,8 +6,8 @@
 from Grasshopper.Kernel.GH_RuntimeMessageLevel import Error
 from Grasshopper.Kernel.GH_RuntimeMessageLevel import Warning
 
-from compas.scene import Scene
 from compas_timber.parts.beam import Beam as ctBeam
+from compas.scene import Scene
```

Comment:
why?

Reply:
I dont think I changed anything here, not intentionally anyways